src/c++/perf_analyzer/docs/llm.md

```diff
@@ -0,0 +1,136 @@
+            ],
+            "STREAM": [
+                true
+            ]
+        }
+    ]
+}
+' > prompts.json
+```
+
+#### 3. Run PA
+
+```bash
+perf_analyzer \
+    -m vllm \
+    -i grpc \
+    --async \
+    --streaming \
+    --input-data=prompts.json \
+    --profile-export-file=profile_export.json \
+    --measurement-mode=count_windows \
+    --measurement-request-count=10 \
+    --stability-percentage=999
+```
+
+#### 4. Calculate average time to first token
+
+```bash
+python3 -c 'import json
+f = open("profile_export.json")
+requests = json.load(f)["experiments"][0]["requests"]
+latencies = [r["response_timestamps"][0] - r["timestamp"] for r in requests]
+avg_latency_s = sum(latencies) / len(latencies) / 1000000000
+print("Average time to first token: " + str(avg_latency_s) + " s")'
+
+# Average time to first token: 0.672695455483871 s
+```
+
+#### 5. Repeat steps 3-4 with different prompt lengths to measure effects of initial prompt size (prefill) on first-token latency.
```

Comment:
I think it might be good to have an example chart for the model deployed using vLLM to demonstrate what the result would look like.

Reply:
Added 👍